src/Components/DisplayOneArticle/index.jsx

```diff
@@ -0,0 +1,116 @@
+import { useState, useEffect } from "react";
+import { getArticleById } from "../../utils/api";
+import "./DisplayOneArticle.css";
+import { useParams } from "react-router-dom";
+
+export function DisplayOneArticle() {
+  const [currentArticle, setCurrentArticle] = useState({
+    author: "",
+    title: "",
+    topic: "",
+    body: "",
+    created_at: "",
+    votes: "",
+    article_img_url: "",
+    comment_count: "",
+  });
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const { article_id } = useParams();
+
+  useEffect(() => {
+    setIsLoading(true);
+    getArticleById(article_id)
+      .then(({ article }) => {
+        setIsLoading(false);
+        setIsError(false);
+        setCurrentArticle(article);
+      })
+      .catch(() => {
+        setIsLoading(false);
+        setIsError(true);
+      });
+  }, [article_id]);
+
+  if (isLoading) return <p>Loading...</p>;
+
+  if (isError) return <p>Whops, some error here... please reload the page!</p>;
+
+  return (
+    <div>
+      <h2>Article Details</h2>
+      <form className="form-article">
+        <label htmlFor="author">Author:</label>
+        <input
```

Comment:
The choice for read only input could be confusing for screen readers

Reply:
https://www.w3schools.com/tags/tag_dl.asp
This is a more semantic way to give things labels